src/funcTest/resources/java-shadow-project/settings.gradle

```diff
@@ -13,4 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-rootProject.name = "java-shadow-project"
\ No newline at end of file
+rootProject.name = "java-forked-shadow-project"
```

Comment:
Why do we need to rename this?

Reply:
I assume "GradleUp" is the new home, so not rather no longer a "fork".

Reply:
Yeah, it's the transferred instead of the fork.